iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTableUtil.java

```diff
@@ -458,6 +458,52 @@ public static List<DeleteFile> getDeleteFiles(Table table, String partitionPath)
         t -> ((PositionDeletesScanTask) t).file()));
   }
 
+  public static float getFileSizeRatio(Table table, String partitionPath, long fileSizeInBytesThreshold) {
```

Comment:
The name of the method does not show what ratio it, as it does nothing with the file actual size, rather it check the number of files. 
Like e.g. getUncompactedRatio would be better.

Reply:
> it does nothing with the file actual size, rather it check the number of files.
The method uses the actual file sizes, it checks for each file if its actual size is above or below the threshold and returns `(number of files whose size is below threshold) / (total number of files)`.
I think `getUncompactedRatio` is fine, I will rename like that.